ocpkg

```diff
@@ -532,10 +532,9 @@ trap quit_trap  INT HUP QUIT TERM
 
 # SECTION 4: Function Definitions
 
-is_x68_64_trusty() {
-OSREL=$(sed -n 's/DISTRIB_CODENAME=\(.*\)/\1/p' /etc/lsb-release);
+is_x68_64() {
 ARCH=$(uname -m);
-if [ "$OSREL" == "trusty" -a "$ARCH" == "x86_64" ]; then
+if [ "$ARCH" == "x86_64" ]; then
   return 0
```

Comment:
so if ARCH equals x86_64, then is_x86_64 returns a zero?    Is this the bash convention where 0 means "error free" and non-zero means "there is an error"?